ckanext/cfpb_extrafields/controllers/digimport.py

```diff
@@ -0,0 +1,47 @@
+try:
+    from cStringIO import StringIO
+except ImportError:
+    from stringIO import StringIO
+import json
+import re
+import urllib
+
+from ckan.plugins.toolkit import BaseController, get_action, redirect_to, render, request, response, ValidationError
+
+from ckanext.cfpb_extrafields.digutils import make_rec
+
+def make_name(title):
+    return re.sub(r"[^a-zA-Z0-9]+", "-", title).lower()
+
+def upload_rec(rec):
+    result = get_action("package_create")({}, rec)
+    return result
+
+class ImportController(BaseController):
+    def index(self, group):
+        errors = request.params.get("errors", [])
+        if errors:
+            try:
+                errors = json.loads(errors)
+            except json.JSONDecodeError:
+                errors = ["Unknown errors detected"]
```

Comment:
Perhaps do both?  I guess this is a very unlikely scenario, but including the info on where the JSON decode error could be useful if we actually need to debug something.